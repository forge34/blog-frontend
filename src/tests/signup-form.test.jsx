import { render } from "@testing-library/react";
import SignupForm from "../components/signup-form";
import userEvent from "@testing-library/user-event";
import { MemoryRouter } from "react-router-dom";

describe("Signup form test", () => {
  it("should match snapshot", () => {
    const {form} = render(
      <MemoryRouter>
        <SignupForm></SignupForm>
      </MemoryRouter>,
    );

    expect(form).toMatchSnapshot()
  });
});
