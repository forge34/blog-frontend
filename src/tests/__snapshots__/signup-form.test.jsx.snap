// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Signup form test > should match snapshot 1`] = `undefined`;
